### PR TITLE
Added backup and restore script for vault secrets

### PR DESCRIPTION
**Pull request by fgimenez**

### CODE REVIEW

File: bin/secrets/backup.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set +x
+
+if [ -z "$1" ]
+then
+    echo "Secrets tree path given as first argument, exiting"
```
Comment (elopio) [inline bin/secrets/backup.sh]: You mean: *No* secrets tree ...

File: bin/secrets/restore.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set +x
+
+if [ -z "$1" ]; then
+    echo "Secrets tree path given as first argument, exiting"
```
Comment (elopio) [inline bin/secrets/restore.sh]: *No* secrets tree...

File: bin/secrets/restore.sh [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+else
+    SECRETS_TREE_PATH="${1}"
+fi
+ENVIRONMENT=${2:-remote}
+
+. ./bin/secrets/common.sh
+
+remove_base_dir(){
+    target_dir="$1"
+    echo ${target_dir#${SECRETS_TREE_PATH}/secret}
```
Comment (elopio) [inline bin/secrets/restore.sh]: I'm failing to understand how this removes the base dir. It's only an echo command, isn't it?

**Comment by elopio** [comment]
:+1: 
Just a couple of message corrections.

File: bin/secrets/restore.sh (hunk 3 above)
Comment (fgimenez) [inline bin/secrets/restore.sh]: renamed to strip_, it's a much better name given the intention

**Pull request merged**